katalog/curator-s3/action.yml

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2020 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+actions:
+  1:
+    action: snapshot
+    description: >-
+      Snapshot logstash- prefixed indices older than ${UNIT_COUNT:1} days (based on index
+      creation_date) with the default snapshot name pattern of
+      'curator-%Y%m%d%H%M%S'.  Wait for the snapshot to complete.  Do not skip
+      the repository filesystem access check.  Use the other options to create
+      the snapshot.
+    options:
+      # the repository name must match with the name of the s3 created task snapshot
+      repository: ${SNAPSHOT_S3_TASK:pippo-s3}
```

Comment:
I saw that in many identical files we set a default, so I thought to put it, just to have evidence that you didn't set a name when the error is printed. This can help to make the debug process quick and avoid to lose time.

Reply:
ah, ok. Then maybe it's better to change the default value to something a little more informative than `pippo-s3`?

Reply:
`pippo-s3` is so cool 🤔